docs/example_ir.ipynb

```diff
@@ -0,0 +1,330 @@
+{
```

Comment:
<p>Maybe you could edit the title with something prompting to "Database example"...and a link-ref to your work of course!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes, I think we could add something about database in the title. E.g., `A simple IR for modeling a database query language`.